src/reference/chat/data.json

```diff
@@ -658,11 +658,20 @@
       }
     },
     "PurgeMessage": {
-      "description": "Sent when a user's messages are purged.",
+      "description": "Sent when a user's messages are purged. Note: The moderator object is only set when a user is purged from in chat. The ban event triggers a purge but the object is not defined.",
```

Comment:
Yea. For consistency, it would be good, I agree. A little bit of a pain since bans are in backend and don't have ready access to the chat author object, but worth it. It'd be super easy for a developer to not test or be aware of the PurgeMessage sent during bans and result in a crash on their end.

For now, this documentation is fine, though.